app/models/drupal_content_type_map.rb

```diff
@@ -29,9 +29,9 @@ def cartographer_notes
   end
 
   def license
-    l = "<a href='http://creativecommons.org/publicdomain/zero/1.0/'>Public Domain</a>" if field_license_value == 'publicdomain'
-    l = "<a href='http://creativecommons.org/licenses/by/3.0/'>CC-BY</a>" if field_license_value == 'cc-by'
-    l = "<a href='http://creativecommons.org/licenses/by-sa/3.0/'>CC-BY-SA</a>" if field_license_value == 'cc-by-sa'
+    l = "<a href='https://creativecommons.org/publicdomain/zero/1.0/'>Public Domain</a>" if field_license_value == 'publicdomain'
+    l = "<a href='https://creativecommons.org/licenses/by/3.0/'>CC-BY</a>" if field_license_value == 'cc-by'
+    l = "<a href='https://creativecommons.org/licenses/by-sa/3.0/'>CC-BY-SA</a>" if field_license_value == 'cc-by-sa'
```

Comment:
@Onyinye91-ctrl Thanks for changing these, even though this isn't what the original issue was asking for, I think this will still work.